src/prefect/runner/submit.py

```diff
@@ -94,62 +120,85 @@ async def _submit_flow_to_runner(
         )
         response.raise_for_status()
 
-        flow_run_id = response.json()["flow_run_id"]
+        flow_run_id = uuid.UUID(response.json()["flow_run_id"])
 
-        return uuid.UUID(flow_run_id)
+        return await client.read_flow_run(flow_run_id)
```

Comment:
Based on the return value of `submit_to_runner` maybe we should only return the UUID id.

Reply:
added!